sklearn/decomposition/nmf.py

```diff
@@ -842,8 +842,13 @@ class NMF(BaseEstimator, TransformerMixin):
     solver : 'pg' | 'cd'
         Numerical solver to use:
         'pg' is a (deprecated) Projected Gradient solver.
+
+        .. versionchanged:: 0.17
```

Comment:
I think I commented on this before. I would do just
```
.. versionadded:: 0.17
    Coordinate Descent solver
```
or something like that, and list the two in the beginning without any version info.